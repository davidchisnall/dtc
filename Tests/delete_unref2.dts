/dts-v1/;

/ {
	/omit-if-no-ref/ qux0: qux@0 { };
	/omit-if-no-ref/ qux1: qux@1 { };
	/omit-if-no-ref/ qux@2 { };
	qux@3 { };

	quux@0 {
		test-property = <&{/qux@1}>;
	};
};
