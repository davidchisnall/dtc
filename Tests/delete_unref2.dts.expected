/dts-v1/;

/  {

	qux@1 {

		phandle = <0x1>;
	};
	qux@3 {

	};
	quux@0 {

		test-property = <0x1>;
	};
};
